PROPOSALCOPY.MD

```diff
@@ -0,0 +1,36 @@
+## Proposal to deploy & fund the bot for depositing buffered ether and refund ether spent on gas fee
+
+User's ether is deposited to the Beacon chain staking via depositBufferedEther call on "DepositSecurityModule" smart contract.
```

Comment:
The motivation is required; another thing to mention here is: we _need_ a bot for vulnerability mitigation, at least for the current implementation (one can't read from Deposit Security Council Kafka by hand)